src/os.rs

```diff
@@ -289,6 +292,51 @@ mod imp {
     }
 }
 
+#[cfg(target_os = "openbsd")]
+mod imp {
+    extern crate libc;
+
+    extern "C" {
```

Comment:
getentropy function is exported via libc crate. So redefining the function prototype shouldn't be required